setup.py

```diff
@@ -21,7 +21,7 @@
     author="Netherlands eScience Center",
     author_email="generalization@esciencecenter.nl",
     url="https://github.com/matchms/matchms",
-    packages=find_packages(),
+    packages=find_packages(exclude=['*tests*']),
```

Comment:
This works, but instead of excluding tests dir you could also remove `tests/__init__.py` to get the same effect.

Reply:
I prefer to do this in setup.py as this is related to how we package and distribute it. Do you have a preference?

Reply:
I like less code/config instead of more, but once we start importing test methods in other test module we might need the __init__.py again. So let's go with your suggestion.